File: tutorial/summing.java
import java.util.Scanner;

public class summing {
    public static void main(String[] args) {
        Scanner in = new Scanner(System.in);

        int n = in.nextInt();

        for (int i = 0; i < n; ++i) {
            // sum the ten integers
            int sum = 0;

            for (int j = 0; j < 10; ++j) {
                sum += in.nextInt();
            }

            // print the result
            System.out.println(sum);
        }
    }
}

File: tutorial/summing.py
from sys import stdin

# read the first line of input and convert it to a number
n = int(stdin.next())

# loop through all the remaining lines in the input
for line in stdin:
    # convert all the number strings to integers
    numbers = map(int, line.split())

    # find the sum and print the result
    print sum(numbers)
